### PR TITLE
Monitoring app (Grafana/Prometheus)

### PR DESCRIPTION
**Pull request by maxhollmann**
This isn't complete yet, but I wanted to open a PR early to see if this is something you would consider merging at all. It's just a simple Grafana/Prometheus setup tracking metrics from the [RPi system](https://hub.docker.com/r/prom/node-exporter), [bitcoind](https://hub.docker.com/r/jvstein/bitcoin-prometheus-exporter), and [lnd](https://github.com/lightninglabs/lndmon).

What do you think?

### Todo

- [ ] Website, description etc in `registry.json`
- [ ] Screenshots for App store
- [ ] App icon
- [ ] Organize dashboards
- [ ] Better Home dashboard

### CODE REVIEW

**Comment by lukechilds** [comment]
Woah this is really cool!

Unfortunately we can't allow apps to have filesystem access outside of the data directory for security reasons. Even readonly, it means the app has access to all Tor/Bitcoin/Lightning/PGP/SSH keys etc.

So we can't do this:

```yaml
  node_exporter:
    image: prom/node-exporter:v1.1.2
    volumes:
      - /proc:/host/proc:ro
      - /sys:/host/sys:ro
      - /:/rootfs:ro
```

But the Bitcoin/LND monitoring is an awesome idea, love it!

**Comment by maxhollmann** [comment]
Right, makes sense, I'll see if it can do anything useful without those binds or remove it if not.

Another question: I'm not sure what to use for author, website, logo etc as different pieces of software are working together for this app. Do you have any suggestions?

File: apps/monitoring/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      - bitcoin_exporter
+      - lndmon
+    restart: on-failure
+
+  node_exporter:
+    image: prom/node-exporter:v1.1.2
+    volumes:
+      - /proc:/host/proc:ro
+      - /sys:/host/sys:ro
+      - /:/rootfs:ro
```
Comment (AaronDewes) [inline apps/monitoring/docker-compose.yml:45]: This is a huge security issue, while proc and sys are already not allowed, this is the biggest issue, why is this needed?

**Comment by mayankchhabra** [comment]
Hey @maxhollmann! With Umbrel 0.5, we moved the all the Umbrel apps to their own dedicated repository — (getumbrel/umbrel-apps)[https://github.com/getumbrel/umbrel-apps]. With this change, new apps (and app updates) are published in the Umbrel App Store in realtime as soon as the PRs are merged.

I'm closing this PR for now. In case you're still interested to ship Grafana/Prometheus, please feel free to open a PR at the new repo. Cheers!
